beta-src/src/enums/map/variants/classic/Province.ts

```diff
@@ -0,0 +1,141 @@
+
+When interacting with the game API, it is also common to work
+with province IDs from the API. It is guaranteed that a 
+provinceID numerically or stringwise matches the territoryID
+of the root territory (i.e. non-special-coast territory) of the 
+province.
+
+Likewise, it is guaranteed that the Province enum for playable
+provinces is stringwise equal to the Territory enum for
+the root territory of that province.
+
+However, we should try not to rely too heavily this mixing
+up of these types if possible.
+
+See GameStateMaps.ts for converting Provinces to/from IDs.
+See ProvincesMapData for getting detailed data about the properties
+of a province and what Territories it has within it.
+*/
+
+enum Province {
+  ADRIATIC_SEA = "ADRIATIC_SEA",
+  AEGEAN_SEA = "AEGEAN_SEA",
+  ALBANIA = "ALBANIA",
+  ANKARA = "ANKARA",
+  APULIA = "APULIA",
+  ARMENIA = "ARMENIA",
+  BALTIC_SEA = "BALTIC_SEA",
+  BARENTS_SEA = "BARENTS_SEA",
+  BELGIUM = "BELGIUM",
+  BERLIN = "BERLIN",
+  BLACK_SEA = "BLACK_SEA",
+  BOHEMIA = "BOHEMIA",
+  BREST = "BREST",
+  BUDAPEST = "BUDAPEST",
+  BULGARIA = "BULGARIA",
+  BURGUNDY = "BURGUNDY",
+  CHANNEL_1 = "CHANNEL_1",
+  CLYDE = "CLYDE",
+  CONSTANTINOPLE = "CONSTANTINOPLE",
+  DENMARK = "DENMARK",
+  EASTERN_MEDITERRANEAN = "EASTERN_MEDITERRANEAN",
+  EDINBURGH = "EDINBURGH",
+  ENGLISH_CHANNEL = "ENGLISH_CHANNEL",
+  FINLAND = "FINLAND",
+  GALICIA = "GALICIA",
+  GASCONY = "GASCONY",
+  GREECE = "GREECE",
+  GULF_OF_BOTHNIA = "GULF_OF_BOTHNIA",
+  GULF_OF_LYONS = "GULF_OF_LYONS",
+  HELIGOLAND_BIGHT = "HELIGOLAND_BIGHT",
+  HOLLAND = "HOLLAND",
+  IONIAN_SEA = "IONIAN_SEA",
+  IRISH_SEA = "IRISH_SEA",
+  KIEL = "KIEL",
+  LIVERPOOL = "LIVERPOOL",
+  LIVONIA = "LIVONIA",
+  LONDON = "LONDON",
+  MARSEILLES = "MARSEILLES",
+  MIDDLE_ATLANTIC = "MIDDLE_ATLANTIC",
+  MOSCOW = "MOSCOW",
+  MUNICH = "MUNICH",
+  NAPLES = "NAPLES",
+  NEUTRAL_1 = "NEUTRAL_1",
+  NEUTRAL_2 = "NEUTRAL_2",
+  NEUTRAL_3 = "NEUTRAL_3",
+  NEUTRAL_4 = "NEUTRAL_4",
+  NEUTRAL_5 = "NEUTRAL_5",
+  NEUTRAL_6 = "NEUTRAL_6",
+  NEUTRAL_7 = "NEUTRAL_7",
+  NEUTRAL_8 = "NEUTRAL_8",
+  NEUTRAL_9 = "NEUTRAL_9",
+  NORTH_AFRICA = "NORTH_AFRICA",
+  NORTH_ATLANTIC = "NORTH_ATLANTIC",
+  NORTH_ATLANTIC2 = "NORTH_ATLANTIC2",
+  NORTH_SEA = "NORTH_SEA",
+  NORWAY = "NORWAY",
+  NORWEGIAN_SEA = "NORWEGIAN_SEA",
+  PARIS = "PARIS",
+  PICARDY = "PICARDY",
+  PIEDMONT = "PIEDMONT",
+  PORTUGAL = "PORTUGAL",
+  PRUSSIA = "PRUSSIA",
+  ROME = "ROME",
+  RUHR = "RUHR",
+  RUMANIA = "RUMANIA",
+  SAINT_PETERSBURG = "SAINT_PETERSBURG",
+  SERBIA = "SERBIA",
+  SEVASTOPOL = "SEVASTOPOL",
+  SILESIA = "SILESIA",
+  SKAGERRACK = "SKAGERRACK",
+  SKAGERRACK2 = "SKAGERRACK2",
+  SMYRNA = "SMYRNA",
+  SPAIN = "SPAIN",
+  SWEDEN = "SWEDEN",
+  SYRIA = "SYRIA",
+  TRIESTE = "TRIESTE",
+  TUNIS = "TUNIS",
+  TUSCANY = "TUSCANY",
+  TYROLIA = "TYROLIA",
+  TYRRHENIAN_SEA = "TYRRHENIAN_SEA",
+  UKRAINE = "UKRAINE",
+  UNPLAYABLE_LAND1 = "UNPLAYABLE_LAND1",
```

Comment:
Not suggesting a change here, but: from a mental model POV, are these really provinces?

Reply:
Well, they're drawable regions of the map, so they share a decent amount of the same data fields that playable provinces have. But they aren't moveable-into and don't have unit slots. So, same, and different.